src/PrestaShopBundle/Resources/views/Admin/Improve/Design/MailTheme/Blocks/generate_mails_form.html.twig

```diff
@@ -0,0 +1,97 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% block generateMailsFormBlock %}
+<div class="row justify-content-center">
+  <div class="col-xl-10">
+    {{ form_start(generateMailsForm, {'action': path('admin_mail_theme_generate')}) }}
+    <div class="card">
+      <h3 class="card-header">
+        <i class="material-icons">email</i> {{ 'Generate emails'|trans({}, 'Admin.Design.Feature') }}
+      </h3>
+
+      <div class="card-block row">
+        <div class="card-text">
+
+          <div class="form-group row">
+            <label class="form-control-label">
+              {{ 'Select your email theme'|trans({}, 'Admin.Design.Feature') }}
+            </label>
+            <div class="col-sm">
+              {{ form_errors(generateMailsForm.mailTheme) }}
+              {{ form_widget(generateMailsForm.mailTheme) }}
+            </div>
+          </div>
+
+          <div class="form-group row">
+            <label class="form-control-label">
+              {{ 'Select your language'|trans({}, 'Admin.International.Feature') }}
+            </label>
+            <div class="col-sm">
+              {{ form_errors(generateMailsForm.language) }}
+              {{ form_widget(generateMailsForm.language) }}
+            </div>
+          </div>
+
+          <div class="form-group row">
+            <label class="form-control-label">
+              {{ ps.label_with_help(
+                ('Select the theme you want to overwrite'|trans({}, 'Admin.Design.Feature')),
+              ('Core email templates are stored in the mails folder but can be overriden by theme proper mails folder. Use this option to overwrite mails from a theme.'|trans({}, 'Admin.Design.Help'))
```

Comment:
_Core email templates are stored in the mails folder but can be **overridden** by theme proper mails folder. Use this option to overwrite **emails** from a theme._

Reply:
Done